activejob/lib/active_job/continuable.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module ActiveJob
+  # = Active Job Continuable
+  #
+  # Mix ActiveJob::Continuable into your job to enable continuations.
+  #
+  # See +ActiveJob::Continuation+ for usage. # The Continuable module provides the ability to track the progress of your jobs,
+  # and continue from where they left off if interrupted.
+  #
+  module Continuable
+    extend ActiveSupport::Concern
+
+    CONTINUATION_KEY = "continuation"
```

Comment:
In the past, the keys used by Active Job itself were prefixed with `_aj_`, e.g.:

```ruby
      GLOBALID_KEY = "_aj_globalid"
      # :nodoc:
      SYMBOL_KEYS_KEY = "_aj_symbol_keys"
      # :nodoc:
      RUBY2_KEYWORDS_KEY = "_aj_ruby2_keywords"
```

We probably should do the same here.

Reply:
Now that this is live is it too late to change it?

Reply:
The `_aj_` prefix is used as part of argument serialization and not for [job data keys](https://github.com/rails/rails/blob/c7c72ee6d39667014e7413c1430a0776b52801b2/activejob/lib/active_job/core.rb#L119-L130), so I think the unprefixed name makes more sense here.

Reply:
My bad. I mistakenly assumed that keys was in the arguments. If it's at the top-level, there shouldn't be risks of conflict indeed.